backend/graphql/index.ts

```diff
@@ -90,6 +90,7 @@ const graphQLMiddlewares = {
     duplicateTest: authorizedByAdmin(),
     unarchiveTest: authorizedByAdmin(),
     archiveTest: authorizedByAdmin(),
+    // createClass: authorizedByAllRoles(),
```

Comment:
can you uncomment this
```suggestion
    createClass: authorizedByAllRoles(),
```